### PR TITLE
Add data collection year

### PR DESCRIPTION
**Pull request by ghostdevv**
I noticed it doesn't specify and thought it might help others if they are curious too

### CODE REVIEW

**Review by devinbaeten — CHANGES_REQUESTED** [review]
Thanks for the suggestion. Before I can merge, please update `src/index.html` to say "September 2022" instead of " 2022".

File: src/index.html [hunk 1]
```diff
@@ -486,7 +486,7 @@ <h1>
...
 						<h2>Data Collection</h2>
-						<p>My server network asks BeReal for the latest moments in each region once per second and records new data as it becomes available; This means that my database is always up to date. I started collecting data in early September. As such, I am unable to provide any accurate data from an earlier time. To view my data, you can visit this site; On the dashboard tab, you can see the latest moments, as well as 90 days of history in each region respectively.</p>
+						<p>My server network asks BeReal for the latest moments in each region once per second and records new data as it becomes available; This means that my database is always up to date. I started collecting data in early  2022. As such, I am unable to provide any accurate data from an earlier time. To view my data, you can visit this site; On the dashboard tab, you can see the latest moments, as well as 90 days of history in each region respectively.</p>
```
Comment (ghostdevv) [inline src/index.html]:
```suggestion
						<p>My server network asks BeReal for the latest moments in each region once per second and records new data as it becomes available; This means that my database is always up to date. I started collecting data in early September 2022. As such, I am unable to provide any accurate data from an earlier time. To view my data, you can visit this site; On the dashboard tab, you can see the latest moments, as well as 90 days of history in each region respectively.</p>
```

**Review by devinbaeten — APPROVED** [review]

**Comment by ghostdevv** [comment]
Done @devinbaeten

**Review by devinbaeten — APPROVED** [review]

**Comment by devinbaeten** [comment]
Thanks again!

**Pull request merged**